FlowCrypt/Models/Realm Models/ClientConfigurationRealmObject.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 import RealmSwift
 
 final class ClientConfigurationRealmObject: Object {
-    @Persisted(primaryKey: true) var userEmail: String!
```

Comment:
Have to revert this back because it requires migration

Reply:
We should ideally have a test for upgrading from earlier app. For this, I suppose we would pre-build an app and check it into the repo. It would forever stay the same until it can no longer run the current iOS version we test on.

The test would:
 - open the app
 - set up an account
 - open the most recent app build
 - verify that basic functionality remains functional (eg no missed or bad migrations)

Not sure if realistic - worth a try.

Reply:
Do you still have to update it in this PR? If yes, then mark it as a draft so it wouldn't merge in the meanwhile.

Reply:
Yes I have to revert Realm schema change. I did also that two requests from new issue.